### PR TITLE
Update METADATA.yaml

### PR DESCRIPTION
**Pull request by robinvanopstal**

### CODE REVIEW

File: METADATA.yaml [hunk 1]
```diff
@@ -1,37 +1,29 @@
...
-description: wg-ui-and-a11y Working Group is responsible for AMP's visual components & interactions and AMP's overall accessibility and user experience (including published guidelines for AMP UX). The UI Working Group also has overall responsibility for ensuring AMP is accessible, though each WG is responsible for ensuring accessibility within their areas of responsibility.
+title: Components
+merged: 
+ - ui-and-a11y
+description: wg-components Working Group is responsible for AMP components, the overall health of AMP Pages, analytics features, and integrations with partner technologies. For overall accessibility, this group is also the first point of contact, however each WG is responsbile for ensuring accessibility within their areas of responsibility.
 facilitator:
-  login: nainar
-  name: Naina Raisinghani
+  login: cathyxz
+  name: Cathy Zhu
```
Comment (rsimha) [inline METADATA.yaml:7]:
```suggestion
  login: zhouyx
  name: Yuxuan Zhou
```

Cathy has left the team. Yuxuan is the facilitator.

**Review by rsimha — COMMENTED** [review]
@zhouyx for approval.

**Comment by kristoferbaxter** [comment]
The Google bot is confused by the Github suggestions tool. I'll merge this since I (a Googler with signed CLAs) was the one to merge this suggestion.

File: METADATA.yaml [hunk 2]
```diff
@@ -1,37 +1,29 @@
...
-  - login: sparhami
-    name: Sepand Parhami
-  - login: wassgha
-    name: Wassim Gharbi
 communication:
   - channel: slack
     name: Slack
     content:
-      - item: "UI Working Group members will use `#wg-ui-and-a11y` channel on AMP's Slack ([signup](https://docs.google.com/forms/d/e/1FAIpQLSd83J2IZA6cdR6jPwABGsJE8YL4pkypAbKMGgUZZriU7Qu6Tg/viewform?fbzx=4406980310789882877)) for real-time discussion. The channel is open to anyone, regardless of membership in UI working group."
+      - item: "Components Working Group members will use `#wg-components` channel on AMP's Slack ([signup](https://docs.google.com/forms/d/e/1FAIpQLSd83J2IZA6cdR6jPwABGsJE8YL4pkypAbKMGgUZZriU7Qu6Tg/viewform?fbzx=4406980310789882877) for real-time discussion. The channel is open to anyone, regardless of membership in UI working group."
```
Comment (zhouyx) [inline METADATA.yaml:19]: nit: `regardless of membership in Components working group`

File: METADATA.yaml [hunk 3]
```diff
@@ -1,37 +1,29 @@
...
+  - channel: hangouts 
+    name: Google Hangouts 
+    content:
+      - item: "Components Working Group will have **Sprint planning meetings** once every 2 weeks. To join in please dial into the following [Hangouts Meet room](https://meet.google.com/auo-gesx-tto). You can also duplicate [this](https://calendar.google.com/event?action=TEMPLATE&tmeid=MTIycTY5czdmb3AxamE0NjNmdjZ2cGhtOGtfMjAxOTExMjJUMTkwMDAwWiBuYWluYXJAZ29vZ2xlLmNvbQ&tmsrc=nainar%40google.com&scp=ALL) calendar reminder to remind yourself of this meeting."
```
Comment (zhouyx) [inline METADATA.yaml:23]: We don't have the external sprint planning meeting set up yet. Can we remove this section for now.

**Comment by robinvanopstal** [comment]
@googlebot I fixed it.

**Comment by rsimha** [comment]
@robinvanopstal Sorry about the CLA noise. We recently switched from Google to OpenJS. If you respond to the last comment and sign the CLA, all your PRs should be in good shape.

**Comment by robinvanopstal** [comment]
@googlebot I fixed it.

File: METADATA.yaml [hunk 4]
```diff
@@ -1,37 +1,25 @@
...
-description: wg-ui-and-a11y Working Group is responsible for AMP's visual components & interactions and AMP's overall accessibility and user experience (including published guidelines for AMP UX). The UI Working Group also has overall responsibility for ensuring AMP is accessible, though each WG is responsible for ensuring accessibility within their areas of responsibility.
+title: Components
+merged: 
+ - ui-and-a11y
+description: wg-components Working Group is responsible for AMP components, the overall health of AMP Pages, analytics features, and integrations with partner technologies. For overall accessibility, this group is also the first point of contact, however each WG is responsbile for ensuring accessibility within their areas of responsibility.
 facilitator:
-  login: nainar
-  name: Naina Raisinghani
+  login: zhouyx
+  name: Yuxuan Zhou
```
Comment (rsimha) [inline METADATA.yaml:7]: Yuxuan has left the team. The updated membership can be found at https://github.com/ampproject/wg-components.

**Review by rsimha — APPROVED** [review]
Thanks for picking this up!

File: METADATA.yaml [hunk 5]
```diff
@@ -1,37 +1,25 @@
-title: User Interface & Accessibility
-description: wg-ui-and-a11y Working Group is responsible for AMP's visual components & interactions and AMP's overall accessibility and user experience (including published guidelines for AMP UX). The UI Working Group also has overall responsibility for ensuring AMP is accessible, though each WG is responsible for ensuring accessibility within their areas of responsibility.
+title: Components
+merged: 
+ - ui-and-a11y
+description: wg-components Working Group is responsible for AMP components, the overall health of AMP Pages, analytics features, and integrations with partner technologies. For overall accessibility, this group is also the first point of contact, however each WG is responsbile for ensuring accessibility within their areas of responsibility.
```
Comment (caroqliu) [inline METADATA.yaml:4]:
```suggestion
description: wg-components Working Group is responsible for AMP components, the overall health of AMP Pages, analytics features, and integrations with partner technologies.
```

File: METADATA.yaml [hunk 6]
```diff
@@ -1,37 +1,25 @@
...
   - channel: github
     name: GitHub
     content:
-      - item: "UI Working Group will post **Status Updates** every two weeks as an issue labeled with `Type: Status Update` in this repository."
-      - item: "UI Working Group will post **Announcements and Notices** regarding events as an issue labeled with `Type: Event` in this repository."
-      - item: "UI Working Group will post **Quarterly Roadmap** as an issue labeled with `Type: Roadmap` in this repository."
+      - item: "Components Working Group will post **Status Updates** every two weeks as an issue labeled with `Type: Status Update` in this repository."
+      - item: "Components Working Group will post **Announcements and Notices** regarding events as an issue labeled with `Type: Event` in this repository."
+      - item: "Components Working Group will post **Quarterly Roadmap** as an issue labeled with `Type: Roadmap` in this repository."
```
Comment (caroqliu) [inline METADATA.yaml:25]:
```suggestion
      - item: "Components Working Group will post **Quarterly Roadmap** as an issue labeled with `Type: Roadmap` in this repository."
      - item: "Components Working Group will post **Monthly Office Hours** as an issue labeled with `Type: Meeting` in this repository."
```

File: METADATA.yaml [hunk 7]
```diff
@@ -1,37 +1,25 @@
...
 facilitator:
-  login: nainar
-  name: Naina Raisinghani
+  login: zhouyx
+  name: Yuxuan Zhou
 members:
-  - login: aghassemi
-    name: Ali Ghassemi
-  - login: alanorozco
-    name: Alan Orozco
```
Comment (caroqliu) [inline METADATA.yaml:10]: Please keep Alan here.

File: METADATA.yaml [hunk 8]
```diff
@@ -1,37 +1,25 @@
...
-    name: Andrew Watterson
-  - login: cathyxz
-    name: Cathy Zhu
-  - login: cvializ
-    name: Carlos Vializ
-  - login: danielcyoung
-    name: Daniel Young
-  - login: joneleven
-    name: Jonathan Leventhal
+  - login: micajuine-ho
```
Comment (caroqliu) [inline METADATA.yaml:9]:
```suggestion
  - login: caroqliu
    name: Caroline Liu
  - login: dmanek
    name: Dhruv Manek
  - login: micajuine-ho
```

**Review by caroqliu — APPROVED** [review]

File: METADATA.yaml (hunk 4 above)
Comment (caroqliu) [reply METADATA.yaml:7]: The facilitator listed for the group at the link above is @micajuine-ho FYI

**Comment by caroqliu** [comment]
@micajuine-ho Can you please review/merge when this is ready? Doesn't look like I have the permissions :)

**Comment by micajuine-ho** [comment]
I don't have write permission either. @kristoferbaxter  Can you take a look?

**Pull request closed without merging**